meteor/client/lib/rundown.ts

```diff
@@ -508,37 +501,26 @@ export namespace RundownUtils {
 					includeDisabledPieces
 				)
 
+				// furthestDuration is used to figure out how much content (in terms of time) is there in the Part
+				let furthestDuration = 0
```

Comment:
What is this variable being used for? I can see it being assigned but I don't see where it is used after.

Reply:
I have no idea, it looks to have been abandoned a while ago. It has been removed now